File: BE/src/main/java/com/ssafy/e106/domain/subscriptionusage/dto/response/SubscriptionUsageReportResponse.java
package com.ssafy.e106.domain.subscriptionusage.dto.response;

import java.util.List;

public record SubscriptionUsageReportResponse(
    SubscriptionUsageReportSummaryResponse summary,
    List<SubscriptionUsageReportInsightResponse> relatedInsights,
    List<SubscriptionUsageReportItemResponse> items) {
}

File: FE/app/src/main/java/com/ssafy/e106/data/dto/subscriptionusage/SubscriptionUsageReportResponse.kt
package com.ssafy.e106.data.dto.subscriptionusage

import kotlinx.serialization.Serializable

@Serializable
data class SubscriptionUsageReportResponse(
    val summary: SubscriptionUsageReportSummaryResponse,
    val relatedInsights: List<SubscriptionUsageReportInsightResponse> = emptyList(),
    val items: List<SubscriptionUsageReportItemResponse> = emptyList(),
)
